components/signup/signupform.tsx

```diff
@@ -21,8 +21,7 @@ import Link from "next/link";
 import { useRouter } from 'next/navigation';
 import { useEffect, useState } from "react";
 import { z } from "zod";
-
-
+import { AiFillEye, AiFillEyeInvisible } from 'react-icons/ai';
```

Comment:
same here too we use lucide icons.

[](https://lucide.dev/icons)

Reply:
ok